doozer/doozerlib/backend/konflux_olm_bundler.py

```
@@ -412,6 +412,33 @@ async def _replace_image_references(self, old_registry: str, content: str, engin
+                    if not isinstance(_img_data, dict):
+                        continue
+                    _delivery = _img_data.get('delivery', {}) or {}
+                    if not _delivery.get('delivery_repo_name_override'):
+                        continue
+                    _repo_names = _delivery.get('delivery_repo_names', [])
+                    if len(_repo_names) == 1:
+                        _img_short = str(_img_data.get('name', '')).rsplit('/', 1)[-1]
+                        _override_short = str(_repo_names[0]).rsplit('/', 1)[-1]
+                        _delivery_override_map[_img_short] = _override_short
+                except Exception:
+                    pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)